MaxText/input_pipeline.py

```diff
@@ -162,7 +168,7 @@ def filter_fn(x):
   # Return multi-host jax.Array prep iterator
   return multihost_gen
 
-
+# this method needs a rewrite in pytorch land (assuming no use of tf.data)
```

Comment:
we can use huggingface datasets library to get C4: https://huggingface.co/datasets/c4